ffi/capi/src/bidi.rs

```diff
@@ -227,15 +227,10 @@ pub mod ffi {
         ///
         /// This is equivalent to calling `paragraph_at()` on `ICU4XBidiInfo` but doesn't
         /// create a new object
-        pub fn set_paragraph_in_text(&mut self, n: usize) -> Result<(), ICU4XError> {
-            let para = self
-                .0
-                .info
-                .paragraphs
-                .get(n)
-                .ok_or(ICU4XError::OutOfBoundsError)?;
+        pub fn set_paragraph_in_text(&mut self, n: usize) -> Option<()> {
```

Comment:
Issue: Not convinced that this is a positive change. This is a setter, not a getter. In Rust, this seems like a place where I would expect panicky behavior. Returning an error seems right.

Reply:
This is the kind of method which would return `bool` in most languages. In our model `bool` is essentially `diplomat_result_void_void` (the struct then only contains `is_ok`); would you be happy with `Result<(), ()>`? Dart already translates `Result<(), ()>` back to `bool`, we could also do this in the other languages. We can even use `bool` instead of `diplomat_result_void_void` on the ABI.

Reply:
I think I'm okay with ABI returning `bool` defined as "true if the operation was successful".